### ISSUE TITLE
App clip not working from your page

### ISSUE THREAD

**Issue by kirill3333** [aheze/AppClipQuickStart#1]
Hey if I open this URL https://aheze.github.io/MyAppClipWebsite/

I see this page not an app clip

![IMG_2170](https://user-images.githubusercontent.com/1804678/137634922-c1f630e3-be73-41d4-9b2e-749b11238d0e.PNG)

**Comment by aheze** [comment]
@kirill3333 yeah, that's because I haven't actually uploaded the sample app to App Store Connect. If you go to https://getfind.app/ then the app clip header should show up.

**Comment by aheze** [comment]
I'll update the README to be more clear.
